apps/mobile/metro.config.js

```diff
@@ -28,7 +28,25 @@ const config = {
     nodeModulesPaths: [
       path.resolve(__dirname, '../../node_modules'),
       path.resolve(__dirname, 'node_modules')
-    ]
+    ],
+    resolveRequest: (context, moduleName, platform) => {
+      // Handle @hylo/presenters subpath imports
```

Comment:
OK, so because presenters need to be accessible to the desktop, it also changes how mobile accesses them? Are there are @hylo/ libraries out of packages that need this treatment or just presenters?

Reply:
only this one changed this way. this is because we load stuff through subpaths like @hylo/presenters/NotificationPresenter, and all the files moved into a src/ directory so they can be built using babel now for the desktop app.